src/CritCore/Translations/Translations.js

```diff
@@ -92,9 +92,11 @@ let translations = {
       userName: 'Nombre de usuario',
       password: 'Ingrese contraseña',
       repeatPassword: 'Repita contraseña',
-      alterEgo: 'Nombre visible / alter ego',
+      alterEgo: 'Nombre visible / Alter ego',
       icon: 'Elija un avatar',
-      submit: 'Crear'
+      submit: 'Enviar',
+      passwordTitle : '7-30 letras & numeros',
+      repeatPasswordTitle : '7-30 letras & numeros'
```

Comment:
Fix number translation and also check structure from english translations.

Reply:
Done.